driver/src/test/scala/com/stratio/sparkta/driver/test/service/RawDataStorageIT.scala

```diff
@@ -30,7 +30,7 @@ import org.scalatest.junit.JUnitRunner
  * This test sucks and I know it...
```

Comment:
I think that you must change the following points (let me know your throughts otherwise).
*You should use "@author annotation" to put your name instead of "Creation by ...". 
*As well, it could be a good idea write a small explanation of what are you doing when you configure the context instead of "This test sucks ..."
*IMHO